src/components/AppSettingsMenu.vue

```diff
@@ -4,6 +4,22 @@
 			{{ t('mail', 'Add mail account') }}
 		</router-link>
 
+		<p v-if="loadingOptOutSettings" class="app-settings">
+			<span class="icon-loading-small"></span>
+			<!-- better label? -->
+			{{ t('mail', 'Collect data..') }}
```

Comment:
My take: "Allow the app to collection data about your interactions. Based on this data, the app will adapt to your preferences. The data will only be stored locally."

Reply:
you omitted the last two sentences on purpose? ;)

Reply:
haha nope, i thought you were explaining to me the first line.

Reply:
nah it's just a bit of text to clarify what "we" do with the data. so users know it's not sent somewhere suspicious but just the Nextcloud instance they're using

Reply:
absolutely